test/e2e/apimachinery/aggregator.go

```diff
@@ -141,7 +139,7 @@ func TestSampleAPIServer(ctx context.Context, f *framework.Framework, aggrclient
 		// kubectl create -f clusterrole.yaml
 		_, err = client.RbacV1().ClusterRoles().Create(ctx, &rbacv1.ClusterRole{
 
-			ObjectMeta: metav1.ObjectMeta{Name: "sample-apiserver-reader"},
+			ObjectMeta: metav1.ObjectMeta{Name: "sample-apiserver-reader-" + namespace},
```

Comment:
Personally I'd make a struct to hold the namespace and these names, set them once, and use them from the struct always, appending in a dozen different places like this is asking for bugs?

Reply:
Updated.